client.py

```diff
@@ -643,14 +643,14 @@
 		ewutils.logMsg("Found neighbors for poi {}: {}".format(poi.id_poi, ewcfg.poi_neighbors[poi.id_poi]))
 
 
-	for id_poi in ewcfg.landmark_pois:
-		ewutils.logMsg("beginning landmark precomputation for " + id_poi)
-		poi = ewcfg.id_to_poi.get(id_poi)
-		ewmap.landmarks[id_poi] = ewmap.score_map_from(
-			coord_start = poi.coord,
-			user_data = fake_observer,
-			landmark_mode = True
-		)
+	# for id_poi in ewcfg.landmark_pois:
+	# 	ewutils.logMsg("beginning landmark precomputation for " + id_poi)
+	# 	poi = ewcfg.id_to_poi.get(id_poi)
+	# 	ewmap.landmarks[id_poi] = ewmap.score_map_from(
+	# 		coord_start = poi.coord,
+	# 		user_data = fake_observer,
+	# 		landmark_mode = True
+	# 	)
```

Comment:
this will work again when we replace the entries in ewcfg.landmark_pois with ones that still exist on the map

Reply:
(use pois that are as far removed from each other as possible)